dev_guide/deployments.adoc

```diff
@@ -552,6 +552,19 @@ pod will have the following properties:
 * The hook failure policy is `Abort`, meaning the deployment will fail if the hook fails.
 * The hook pod will inherit the `data` volume from the deployment configuration pod.
 
+==== Using the command line
```

Comment:
Please add the following anchor above the title:
[[deployments-using-the-command-line]]